src/theory/arith/proof_checker.cpp

```diff
@@ -407,6 +408,21 @@ Node ArithProofRuleChecker::checkInternal(ProofRule id,
       }
       return args[0];
     }
+    case ProofRule::ARITH_POLY_NORM_REL:
+    {
+      Assert(children.size() == 1);
```

Comment:
Can you also guard that children[0] is indeed an equality with the correct form?

Reply:
Done!